src/en/Components/BLE/ble_stack/ble_stack.rst

```diff
@@ -592,7 +592,7 @@ API参考
     };
 
 此数据结构用来填充扫描参数，
-type：为扫描类型有2种类型BT_LE_SCAN_TYPE_ACTIVE（0x01）、BT_LE_SCAN_TYPE_PASSIVE(0x00)。
+type：为扫描类型有2种类型BT_LE_SCAN_TYPE_ACTIVE（0x01）, BT_LE_SCAN_TYPE_PASSIVE(0x00)。
```

Comment:
I wouldn't change the original Chinese documentation unless really necessary.

Reply:
Fixed, some of the 、characters were left in translation in place of commas, I'll be more careful using Find+Replace in the future